be/tool/main.c

```diff
@@ -80,6 +186,35 @@ static void be_recovery_run(char *path)
 	m0_betool_m0_fini();
 }
 
+static void scan(char *path, char *what)
+{
+	struct m0_be_ut_backend        ut_be = {};
+	struct m0_be_domain_cfg        cfg = {};
+	struct m0_be_domain           *dom;
+	char                           location[0x100] = {};
+	int                            rc;
+
+	m0_node_uuid_string_set(NULL);
+	m0_betool_m0_init();
+	m0_be_ut_backend_cfg_default(&cfg);
+	if (path != NULL) {
+		snprintf((char *)&location, ARRAY_SIZE(location),
+			 "linuxstob:./%s", path);
+		cfg.bc_stob_domain_location = (char *)&location;
+		M0_LOG(M0_ALWAYS, "location=%s", cfg.bc_stob_domain_location);
+	}
+	rc = m0_be_ut_backend_init_cfg(&ut_be, &cfg, false);
+	M0_ASSERT_INFO(rc == 0, "rc=%d", rc);
```

Comment:
These code are duplicated with those in be_recovery_run().
Can we fan this out?

Reply:
In be_recovery_run() we are calling m0_be_ut_backend_init_cfg(), whereas in be_log_resize() we are calling m0_be_ut_backend_log_resize()  
only below code is common, 
`        m0_betool_m0_init();
        m0_be_ut_backend_cfg_default(&cfg);

        if (path != NULL) {
                snprintf((char *)&location, ARRAY_SIZE(location),
                         "linuxstob:%s", path);
                cfg.bc_stob_domain_location = (char *)&location;
        }
`

should we move it to new function ?